src/elements/Arrow/Arrow.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import css from './Arrow.module.css';
+
+export interface ArrowProps {
+  /**
+   * CSS Classname to apply to the arrow
+   */
+  className?: string;
+
+  /**
+   * The direction of the arrow
+   */
+  direction?: 'up' | 'right' | 'down' | 'left';
+}
```

Comment:
@amcdnl was thinking about this.. I wonder if we should do a `size` prop here and pass it into the width/height in the svg .. and then maybe use `currentColor` as the fill by default which can get overridden with a className. I didn't quite like how I added an extra className to the default icons for the Tree component

Reply:
It should have inherited it from the CSS in the button.